.github/workflows/ci-zk-toolbox-reusable.yml

```diff
@@ -113,61 +113,89 @@ jobs:
           ci_run sleep 5
 
-      - name: Run integration tests
-        run: |
-          ci_run zk_supervisor test integration --ignore-prerequisites --verbose
-
-      - name: Init external node server
+      - name: Init external node (Rollup)
         run: |
           ci_run zk_inception external-node configs --db-url=postgres://postgres:notsecurepassword@postgres:5432 \
-          --db-name=zksync_en_localhost_era --l1-rpc-url=http://reth:8545
-          ci_run zk_inception external-node init --ignore-prerequisites
+          --db-name=zksync_en_localhost_era --l1-rpc-url=http://reth:8545 --chain chain_rollup
+          ci_run zk_inception external-node init --ignore-prerequisites --chain chain_rollup
 
-      - name: Run recovery tests (from snapshot)
+      - name: Create and initialize Validium chain
         run: |
-          ci_run zk_supervisor test recovery --snapshot --ignore-prerequisites --verbose
-            
-      - name: Run recovery tests (from genesis)
+          ci_run zk_inception chain create \
+          --chain-name chain_validium \
+          --chain-id sequential \
+          --prover-mode no-proofs \
+          --wallet-creation localhost \
+          --l1-batch-commit-data-generator-mode validium \
+          --base-token-address 0x0000000000000000000000000000000000000001 \
+          --base-token-price-nominator 1 \
+          --base-token-price-denominator 1 \
+          --set-as-default false \
+          --ignore-prerequisites
+
+          ci_run zk_inception chain init \
+          --deploy-paymaster \
+          --l1-rpc-url=http://reth:8545 \
+          --server-db-url=postgres://postgres:notsecurepassword@postgres:5432 \
+          --server-db-name=zksync_server_localhost_validium \
+          --prover-db-url=postgres://postgres:notsecurepassword@postgres:5432 \
+          --prover-db-name=zksync_prover_localhost_validium \
+          --port-offset 2000 \
+          --chain chain_validium
+
+      - name: Init external node (Validium)
         run: |
-          ci_run zk_supervisor test recovery --ignore-prerequisites --verbose
-  
-      - name: Run external node server
+          ci_run zk_inception external-node configs --db-url=postgres://postgres:notsecurepassword@postgres:5432 \
+          --db-name=zksync_en_localhost_era --l1-rpc-url=http://reth:8545 --chain chain_validium
+          ci_run zk_inception external-node init --ignore-prerequisites --chain chain_validium
+
+      - name: Run server on Validium chain
         run: |
-          ci_run zk_inception external-node run --ignore-prerequisites &>external_node.log &
+          ci_run zk_inception server --ignore-prerequisites --chain chain_validium &>server_validium.log &
           ci_run sleep 5
 
-      - name: Run integration tests en
+      - name: Fund test wallets
         run: |
-          ci_run zk_supervisor test integration --ignore-prerequisites --verbose --external-node
+          ci_run zk_supervisor test fund
```

Comment:
let's have it as a part of test run

Reply:
Addressed in https://github.com/matter-labs/zksync-era/pull/2594/commits/9e1e02fac18037ac4d46c156942fb9a269cfe51c